### PR TITLE
spaces, min amount witdraw and add modals

### PR DESCRIPTION
**Pull request by Pasha8914**
# Pull request template

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/Contribution) guide
- [ ] branch rebased on `develop`
- [ ] styleguide check `npm run validate`
- [ ] good naming
- [ ] keep simple
- [ ] video or screenshot attached
- [ ] testing instruction attached
- [ ] check your PR once again

### CODE REVIEW

**Comment by noxonsu** [comment]
почемуто перезалит пул https://github.com/swaponline/swap.react/pull/1022

**Comment by noxonsu** [comment]
@DariaPurtova сделай пруфрид этих надписей

File: shared/components/modals/WithdrawModal/WithdrawModal.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ import { FormattedMessage } from 'react-intl'
 
 
 const minAmount = {
-  eth: 0.05,
-  btc: 0.004,
+  eth: 0.005,
+  btc: 0.001,
```
[the remark this answers is not in the document]
Comment (noxonsu) [reply shared/components/modals/WithdrawModal/WithdrawModal.js]: 0.01 думаю

**Comment by DariaPurtova** [comment]
> @DariaPurtova сделай пруфрид этих надписей
> 

- Make sure the wallet you're sending the funds to supports BTC

- (Please note) Miners' fee is 0.001 BTC. Your balance must exceed this sum to perform transactions
или
Your balance must exceed 0.001 BTC to perform transactions. This amount is equivalent to miners' fee

Предлагаю:
- фразу "Make sure the wallet you're sending the funds to supports BTC" поместить в "?" после Address. A в поле, куда надо вставить адрес, написать (Enter) BTC wallet address
- второе поле: Enter the amount (you want) to withdraw/transfer. You have 0.0054 BTC

screen shot 2018-11-27 at 3 57 49 pm

**Pull request merged**